spring-beans/pom.xml

```diff
@@ -54,6 +54,12 @@
       <artifactId>zipkin-reporter</artifactId>
       <version>${project.version}</version>
```

Comment:
Exclusion here (as in `brave/pom.xml`)?

```
 <exclusions>
        <exclusion>
          <groupId>io.zipkin.zipkin2</groupId>
          <artifactId>zipkin</artifactId>
        </exclusion>
      </exclusions>
```

Reply:
good eye, I was about 5 minutes from finding this out myself ;)